### ISSUE TITLE
What training strategy is used for sentence pair regression?

### ISSUE THREAD

**Issue by timpal0l** [ThilinaRajapakse/simpletransformers#695]
Hi,

when finetuning a transformer for a sentence pair regression task, e.g the sts-b corpus. - https://github.com/ThilinaRajapakse/simpletransformers/blob/master/docs/_docs/10-sentence-pair-classification.md

What training strategy is used? ( https://medium.com/swlh/solving-sentence-pair-tasks-using-simple-transformers-2496fe79d616 )

I was looking at the sentence-bert paper - https://arxiv.org/pdf/1908.10084.pdf 
and see that they present 3 different types of training techniques when finetuneing to sentence pairs.

Do we concatenate the sentence embeddings u and v with the element-wise difference |u−v| and multiply it with the trainable weight? 

Or is two embeddings generated, one for each sentence, and then the cosine similarity is computed. And we minimize the loss between the cosine similarity and the labels?

Thanks!

**Comment by ThilinaRajapakse** [comment]
Nothing too fancy. It's just concatenating the two sentences with a separator token in between.

**Comment by timpal0l** [comment]
> Nothing too fancy. It's just concatenating the two sentences with a separator token in between.

Okay thanks! Where in the source code can I see this? :) Did you notice any difference when adding the seperator token?

**Comment by ThilinaRajapakse** [comment]
You can find it [here](https://github.com/ThilinaRajapakse/simpletransformers/blob/master/simpletransformers/classification/classification_utils.py#L81).

Whether or not the separator is added depends on the model (and whether the original implementation did).
